tests/unit/library/test_block.py

```diff
@@ -25,7 +25,8 @@ def test_grammar_opener():
     assert str(model) == "open texta"
 
 
-@pytest.mark.skip(reason="not supporting block grammar definitions for this release")
+# TODO(nopdive): Review this exception later -- how should we be going about grammars in blocks overall.
+@pytest.mark.skip(reason="requires review")
```

Comment:
I think we need to revisit the entire concept of blocks, and I am not fundamentally opposed to requiring block openers/closers to be strings. We can push this discussion to a later date -- definitely doesn't need to be resolved in this PR